elf_binary.h

```diff
@@ -45,15 +46,20 @@ class ELFBinary {
     const std::string& runpath() const { return runpath_; }
     const std::string& rpath() const { return rpath_; }
 
-    const Elf_Sym* symtab() const { return symtab_; }
```

Comment:
To fix binaries in place, I removed `const` from this function and add some functions without `const`.  But I think it is not a good idea. Should I make `symtab_mut`, `verneed_mut`, `versym` and so, on?

Reply:
Sounds good!

Reply:
I made ***_mut methods.